src/events/userChanged.js

```diff
@@ -27,6 +27,8 @@ export default async ({ event }) => {
       user: user.id,
     });
     if (!user.profile.fields) return;
+    // return if there is no user with this slackID
```

Comment:
nit - suggest moving this comment to previous line as I believe that's what it is referring to